#request 
string desiredOutput
---
#response
string modifiedOutput